js/app.js

```diff
@@ -13,14 +13,16 @@ console.log('user stated their name was ' + userName);
 //  } else {
 //      alert ( 'hello , + userName +  'welcome to my site');
 //  }
-
 var myName = prompt(userName + ', is my name harlen');
-if (myName === 'yes') {
-  score++;
-  alert('great job' + userName + '!');
-} else {
-  alert('sorry ' + userName + ', that is incorrect.');
-}
+var q1 = function(){
```

Comment:
not necessary to store the function in var - you can just do:
`function q1 () {
}`
Let me know if you have any questions on this!